tests/containers/podman_network.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman network
+# Summary: Test podman network 
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use containers::utils qw(registry_url container_ip);
+
+sub run(){
+
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+
+    record_info('Test', 'Create a new networks named newnet and newnet2');
+    assert_script_run('podman network create newnet');
+    assert_script_run('podman network create newnet2');
+
+    record_info('Test', 'List the created networks');
+    assert_script_run('podman network ls | grep -e newnet -e newnet2');
```

Comment:
I would do the following instead:
```
record_info('ls', script_output('podman ls'));
assert_script_run('podman network exists newnet1');
assert_script_run('podman network exists newnet2');
```

This way you verify the command `ls` and the command `exists`.

Reply:
`podman ls` is not a valid command. Did you mean `podman network ls`?

Reply:
oh yes, sorry, `podman network ls`

Reply:
done